types/w3c-web-usb/w3c-web-usb-tests.ts

```diff
@@ -0,0 +1,72 @@
+// Modified from WebUSB spec: https://wicg.github.io/webusb/
+
+let connectedDevices: USBDevice[] = null;
+
+document.addEventListener('DOMContentLoaded', async () => {
+    const devices = await navigator.usb.getDevices();
+    devices.forEach(handleConnectedDevice);
+});
+
+let button = document.getElementById('request-device');
+button.addEventListener('click', async () => {
+    let device;
+    try {
+        device = await navigator.usb.requestDevice({ filters: [{
+            vendorId: 0xABCD,
+            classCode: 0xFF, // vendor-specific
+            protocolCode: 0x01
+        }]});
```

Comment:
why not add handleConnectedDevice here?

Reply:
was making it close to the example code from spec